### ISSUE TITLE
Downloading C/C++ language components (Windows)

### ISSUE THREAD

**Issue by AgungPN** [microsoft/vscode-cpptools#8309]
why can't I update the C/C++ extension, the process never runs for hours. Even though the connection is smooth

**Comment by michelleangela** [comment]
@AgungPN 
This may be a duplicate of https://github.com/microsoft/vscode-cpptools/issues/8306 where the hosts to the links of the binaries in the extension package is having issues or the links could be blocked by a VPN, firewall, etc.

As a workaround, please try the offline installation package at https://github.com/microsoft/vscode-cpptools/releases/tag/1.7.1

**Comment by sean-mcmanus** [comment]
Fixed with https://github.com/microsoft/vscode-cpptools/releases/tag/1.8.0-insiders3 or 1.8.0 (via using VS Code's "Pre-Release Version" UI or https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools&ssr=false#version-history ).
